community/stress-ng/APKBUILD

```diff
@@ -1,34 +1,24 @@
 # Contributor: Olliver Schinagl <oliver@schinagl.nl>
 # Maintainer: Olliver Schinagl <oliver@schinagl.nl>
 pkgname=stress-ng
-pkgver=0.09.57
+pkgver=0.10.02
 pkgrel=0
-pkgdesc="stress-ng will stress test a computer system in various selectable ways."
+pkgdesc="stress-ng will stress test a computer system in various selectable ways"
 url="https://kernel.ubuntu.com/~cking/stress-ng/"
 arch="all"
 license="GPL-2.0-or-later"
 options="!check" # tests are not portable
-makedepends="
-	linux-headers
-	libaio-dev
-	libbsd-dev
-	lksctp-tools-dev
-	zlib-dev
-"
+makedepends="libaio-dev libbsd-dev linux-headers lksctp-tools-dev zlib-dev"
```

Comment:
please don't single-line this; keep it as a block, so we can more easily sort it and avoid having.
Thanks for spotting the missing sort, indeed, linux comes after the libs.

Reply:
change reverted.